src/components/Helpers.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+
+class Helpers {
+  static valueToString(value) {
+    if (typeof value === 'string' || value instanceof String) {
```

Comment:
Can we get rid of the ` || value instanceof String`?

Reply:
No( This is a common condition to check if a variable is a string.